eth/protocols/eth/handlers.go

```diff
@@ -159,9 +159,11 @@ func AnswerGetBlockBodiesQuery(db kv.Tx, query GetBlockBodiesPacket, blockReader
 	return bodies
 }
 
-type getReceiptsFunc func(context.Context, *chain.Config, kv.Tx, *types.Block, []libcommon.Address) (types.Receipts, error)
+type ReceiptsGetter interface {
```

Comment:
FYI: 
- we also have `turbo/services/interfaces.go`. But define interfaces on consumer side is also good. 
- can use next trick: 
```
var _ evmtypes.IntraBlockState = new(IntraBlockState) // compile-time interface-check
```

Reply:
done in an another file